includes/admin/settings/class-wc-settings-products.php

```diff
@@ -139,6 +139,14 @@ public function get_settings( $current_section = '' ) {
 						'type'    => 'checkbox',
 					),
 
+					array(
```

Comment:
I'm not sold on this being an option in settings. It will just cause confusion having an option and then still having the ability to check and uncheck when refunding. I'd suggest removing the option and going for a filter instead which defaults to checked state.